cl_sii/extras/dj_form_fields.py

```diff
@@ -0,0 +1,72 @@
+"""
+cl_sii "extras" / Django form fields.
+
+"""
+try:
+    import django
+except ImportError as exc:  # pragma: no cover
+    raise ImportError("Package 'Django' is required to use this module.") from exc
+
+from typing import Any, Optional
+
+import django.core.exceptions
+import django.db.models
+import django.db.models.fields
+import django.forms
+from django.utils.translation import gettext_lazy as _
+
+from cl_sii.rut import Rut
+
+
+class RutField(django.forms.CharField):
+
+    """
+    Django form field for RUT.
+
+    * Python data type: :class:`cl_sii.rut.Rut`
+
+    .. seealso::
+        :class:`.dj_model_fields.RutField`
+
+    """
+
+    default_error_messages = {
+        'invalid': _('Enter a valid RUT.'),
+    }
+
+    def __init__(self, *, validated_dv: bool = False, **kwargs: Any) -> None:
+        """
+        :param validated_dv: Boolean that specifies whether the RUT's
+            "digito verificador" should be validated. False by default.
+        """
+
+        self.validated_dv = validated_dv
```

Comment:
Shouldn't `validated_dv` be named `validate_dv`?

Reply:
I agree that it should be `validate_dv` (without the `d` after `validate`), unless there's some precedent to the contrary.